php/class-admin.php

```diff
@@ -397,4 +399,36 @@ public function plugin_activated_notice() {
 			)
 		);
 	}
+
+	/**
+	 * Enable auto updates for the plugin.
+	 *
+	 * @param bool|null $update Whether to update.
+	 * @param object    $item   The update offer.
+	 *
+	 * @return bool
+	 */
+	public function enable_plugin_auto_update( $update, $item ) {
+		if ( 'material-design' === $item->slug ) {
+			return true;
+		}
+
```

Comment:
I thought we were going to make this all opt-out.

Reply:
Confirmed with Ravi this will take place in admin settings